scene/gui/graph_node.cpp

```diff
@@ -1180,6 +1193,23 @@ Vector<int> GraphNode::get_allowed_size_flags_vertical() const {
 	return flags;
 }
 
+void GraphNode::set_slots_focus_mode(Control::FocusMode p_focus_mode) {
+	if (slots_focus_mode == p_focus_mode) {
+		return;
+	}
+	ERR_FAIL_COND((int)p_focus_mode < 1 || (int)p_focus_mode > 3);
+
+	slots_focus_mode = p_focus_mode;
+	if (slots_focus_mode == Control::FOCUS_CLICK) {
```

Comment:
```suggestion
	if (slots_focus_mode == Control::FOCUS_CLICK && selected_slot > -1) {
```
Unless `queue_redraw()` is needed for something else.

Reply:
No, should not be needed if it's already `-1`.